lib/rubocop/extension/generator/generator.rb

```diff
@@ -7,7 +7,7 @@ def initialize(name)
         end
 
         def generate
-          system('bundle', 'gem', '--test=rspec',  name, exception: true)
+          system('bundle', 'gem', '--linter=rubocop', '--test=rspec',  name, exception: true)
```

Comment:
We could also build on this by enabling some `InternalAffairs` cops by default.

For example, `InternalAffairs/InheritDeprecatedCopClass` is probably a good one to enable.